.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+
+name: build
+
+on: push
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby:
+          - '3.0'
+          - '2.7'
+          - '2.6'
+          - '2.5'
+          - '2.4'
+          - truffleruby
```

Comment:
Can this test be set to allow failure like it was with Travis CI?

Reply:
As I mentioned in the previous comment we don't have something like that.

That said, also note that before we were testing (and allowing failure) against `truffleruby-head`. Now my intention is to test against `truffleruby` (stable truffleruby) and require it passes.

The concern and reason why I didn't consider TruffleRuby to be one of the required rubies before was because it takes a significant higher amount of time to run agasint TruffleRuby than CRuby, which made the TravisCI run take a lot of time to finish. Now I'm seeing that GitHub Actions takes in total way way less time to finish, so including TruffleRuby (while still taking more time than CRuby) it doesn't increase to total time by a lot so it's doable.

Reply:
Great! Didn't notice that difference between `truffleruby` and `truffleruby-head`. Good to know!